backend-models/service-tests/sectionedservice-test.txt

```diff
@@ -0,0 +1,551 @@
+$feedbackquestion != null
+$moment != null
+$kind != null
+$link != null
+$visibility != 0
+}
+
+/////////////////////////
+
+You also need to gather all listentries to compare if any updated.
+SQL Query: "SELECT * FROM listentries" save into beforeUpdatedL
+You also new all code examples
+SQL Query: "SELECT * FROM codeexample" save into beforeUpdatedC
+And lastly the list table for the specific course
+SQL Query: "SELECT * FROM list" save into beforeUpdatedList
+
+*******************************************
+ Get updated code examples and listentries
+*******************************************
+
+SQL Query: "SELECT * FROM codeexample" save into afterNewC
+SQL Query: "SELECT * FROM listentries" save into afterNewL
+SQL Query: "SELECT * FROM list" save into afterNewList
+
+*****************
+ Compare results
+*****************
+
+If any new table to codeexample and list, and updated listentries: passed
+If non new table: failed
+If $debug not empty: failed
+
+*****************
+ Service output
+*****************
+
+"codeexample"
+
+----------------------------------
+updateDeadlineTest() line 371:
+----------------------------------
+
+(checklogin) IS TRUE
+AND $ha || $studentTeacher IS TRUE
+
+//// PRE-REQ Values /////
+
+send
+{
+$opt = "UPDATEDEADLINE"
+$deadline != 0
+$relativedeadline != 0
+$link != 0
+}
+
+/////////////////////////
+
+You also need to gather all quizzes to compare if any updated.
+SQL Query: "SELECT deadline, relativedeadline FROM quiz" save into beforeUpdated
+
+*******************************************
+ Get updated deadlines
+*******************************************
+
+SQL Query: "SELECT deadline, relativedeadline FROM codeexample" save into afterUpdated
+
+*****************
+ Compare results
+*****************
+
+If deadline, relativedeadline has been updated: passed
+If deadline, relativedeadline not updated (same as beforeUpdated): failed
+If $debug not empty: failed
+
+*****************
+ Service output
+*****************
+
+"codeexample"
+
+----------------------------------
+updateTabsTest() line 381:
+----------------------------------
+
+(checklogin) IS TRUE
+AND $ha || $studentTeacher IS TRUE
+
+//// PRE-REQ Values /////
+
+send
+{
+$opt = "UPDATETABS"
+$sectid != 0
+$tabs != 0
+}
+
+/////////////////////////
+
+You also need to gather all listentries to compare if any updated.
+SQL Query: "SELECT gradesystem FROM listentries" save into beforeUpdated
+
+*******************************************
+ Get updated listentries
+*******************************************
+
+SQL Query: "SELECT gradesystem FROM listentries" save into afterUpdated
+
+*****************
+ Compare results
+*****************
+
+If gradesystem has been updated: passed
+If gradesystem not updated (same as beforeUpdated): failed
+If $debug not empty: failed
+
+*****************
+ Service output
+*****************
+
+Not able to identify output for listentries
+
+----------------------------------
+updateVersTest() line 390:
```

Comment:
well.. it seems like the same problems repeat them selves, the same apply for the rest of the doc